ee/vellum_ee/workflows/display/nodes/base_node_vellum_display.py

```diff
@@ -30,6 +30,9 @@ def get_display_data(self) -> NodeDisplayData:
     def get_target_handle_id(self) -> UUID:
         return self._get_node_display_uuid("target_handle_id")
 
+    def get_target_handle_id_by_source_node_id(self, source_node_id: UUID) -> UUID:
+        return self.get_target_handle_id()
```

Comment:
Can we add a code comment like:

In the vast majority of cases, nodes will only have a single target handle and can be retrieved independently of the source node. However, in rare cases (such as legacy Merge nodes), this method can be overridden to account for the case of retrieving one amongst multiple target handles on a node.